.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "next/core-web-vitals",
+  "rules": {
+    "@next/next/no-before-interactive-script-outside-document": "off"
```

Comment:
The reason I had to disable [this lint rule](https://nextjs.org/docs/messages/no-before-interactive-script-outside-document) is because Next.js is no longer using `_document.js` if only to support migrating to using `layout.js`, so it's a bit of a false negative. See here: https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#migrating-_documentjs-and-_appjs